File: JAVA/PARCIAL II/EVA07-POO-TrujilloDavid/src/paquete1/Masajista.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package paquete1;

/**
 *
 * @author Asus
 */
public class Masajista extends Persona{
    protected String titulo;
    protected int aniosExp;

    public Masajista(int aniosExp, int id, String nombre, int edad, String titulo) {
        super(id, nombre, edad);
        this.titulo = titulo;
        this.aniosExp = aniosExp;
    }
    
    public String DarMasaje(){
        return "El masajista "+super.nombre+" esta dando un masaje";
    }
    @Override
    public String Concentrarse() {
        return "El masajista "+super.nombre+" se esta concentrando";
    }

    @Override
    public String Viajar() {
        return "El entrenador "+super.nombre+" esta viajando";
    }

    @Override
    public String toString() {
        return super.toString() + " Titulo: "+titulo+" Años Experiencia: "+aniosExp;
    }
    
    
    
}

File: JAVA/PARCIAL II/EVA-POO-TrujilloDavid/EVA-POO-TrujilloDavid/Masajista.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EVA_POO_TrujilloDavid
{
	internal class Masajista : Persona, IMasajista
	{
		private string titulo;
		private int aniosExperiencia;
		public Masajista(int id, string nombre, int edad, string titulo, int anioExperiencia) : base(id, nombre, edad)
		{
			this.titulo = titulo;
			this.aniosExperiencia = anioExperiencia;
		}

		public string Titulo
		{
			get { return titulo; }
			set { titulo = value; }
		}
		public int AniosExperiencia
		{
			get { return aniosExperiencia; }
			set { aniosExperiencia = value; }
		}
		public override string Concentrarse()
		{
			return "El masajista " + base.Nombre + " esta concentrandose";
		}

		public string DarMasaje()
		{
			return "El masajista " + base.Nombre + " esta dando un masaje";
		}

		public override string Viajar()
		{
			return "El masajista " + base.Nombre + " esta viajando";
		}

		public override string ToString()
		{
			return base.ToString() + " Titulo: "+titulo + " Años Experiencia: "+aniosExperiencia;
		}
	}
}
